test/functional/commands/keyboard/keyboard-e2e-specs.js

```diff
@@ -18,17 +18,30 @@ const PACKAGE = 'io.appium.android.apis';
 const TEXTFIELD_ACTIVITY = '.view.TextFields';
 const KEYEVENT_ACTIVITY = '.text.KeyEventText';
 
-let defaultAsciiCaps = Object.assign({}, APIDEMOS_CAPS, {
+const defaultAsciiCaps = Object.assign({}, APIDEMOS_CAPS, {
   newCommandTimeout: 90,
   appPackage: PACKAGE,
   appActivity: TEXTFIELD_ACTIVITY
 });
 
-let defaultUnicodeCaps = Object.assign({}, defaultAsciiCaps, {
+const defaultUnicodeCaps = Object.assign({}, defaultAsciiCaps, {
   unicodeKeyboard: true,
   resetKeyboard: true
 });
 
+async function ensureUnlocked (driver) {
```

Comment:
Is there a way to disable emulator lock while setting it on travis?

Reply:
Not that I have ever found, no.